app/models/ability.rb

```diff
@@ -6,6 +6,7 @@ class Ability
   def initialize(user)
     can :read, Map
     can :read, Location
+    can :read, Account
```

Comment:
Any reason for a public read access to the account? Would remove this otherwise

Reply:
No there isn't.  I understood it like this means that the logged in user has read access to the Account?

Reply:
This would be in the second block below `if user.present?`

Reply:
TIL - updating!